### PR TITLE
add dev server

### PR DESCRIPTION
**Pull request by stephenkingsley**
Add a simple example and a dev server discussed in #847.

Now, you can follow me step by step:
 - git clone
 - cd preact/
 - npm install
 - npm run dev
 - open `localhost:3000` in chrome

Congratulations! You can see the simple example now!

### CODE REVIEW

File: example/devServer.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+	if (err) throw err;
+	if (stdout) {
+		console.log(`npm run build:dist --- ${stdout}`);
+	}
+	if (stderr) {
+		console.log(`npm run build:dist --- ${stderr}`);
+	}
+});
+
+watch([srcPath, examplePath], {
```
Comment (developit) [inline example/devServer.js]: Would it make more sense to use `webpack-dev-server` for this?

Comment (stephenkingsley) [reply example/devServer.js]: sure, I think use `webpack-dev-server` is simple! I will fix it.

**Comment by stephenkingsley** [comment]
@developit I fix it, please review again!

**Comment by stephenkingsley** [comment]
@developit Can merge this pr into master?

**Comment by developit** [comment]
Hiya - I'd like to have this use `webpack-dev-server` and build preact from source.  I can do that while merging though, just need some time! :)

**Comment by marvinhagemeister** [comment]
We do have a tiny demo app in `master` thanks to #1302 :tada:

**Pull request closed without merging**